packages/zipkin/index.d.ts

```diff
@@ -150,8 +150,6 @@ declare namespace zipkin {
     record: (rec: any) => void;
   }
 
-  const serializeSpan: () => void;
-
   class ExplicitContext {
```

Comment:
if someone wants to add typescript for model.Span or model.Endpoint, they could as well.. we can also wait until we add the v2 api (as most won't work with model types directly)